README.md

```diff
@@ -18,3 +18,9 @@ Used 3rd party technologies and assets:
 - CSS framework [Bulma](https://github.com/jgthms/bulma)
 - [TypeScript](https://github.com/Microsoft/TypeScript)
 - [Webpack](https://github.com/webpack/webpack)
+
+## How to develop
+
+- install docker
+- make
```

Comment:
Unclear what the correct sync tax should be here. @pwerry can you help me out.

My understanding is that Makefile typically starts with some variable definitions which are then followed by a set of target entries for building specific targets (typically .o & executable files in C and C++, and . class files in Java) or executing a set of commands associated with a target label.

Reply:
I haven't used a makefile in years... I'm glad @pwerry is the expert here.